### PR TITLE
Fix icon mapping time

### PR DESCRIPTION
**Pull request by sjmiles**
Fixing #10

### CODE REVIEW

File: iron-iconset-svg.html [hunk 1]
```diff
@@ -63,14 +63,6 @@
       },
 
       /**
-       * Array of fully-qualitifed icon names in the iconset.
-       */
-      iconNames: {
```
Comment (cdata) [inline iron-iconset-svg.html]: While this change is fine, it's notable in that it "breaks" the API for this element. What are your thoughts re: versioning?

Comment (sjmiles) [inline iron-iconset-svg.html]: I have no magic sauce. It's a _breaking change_, but I can't avoid it. We can avoid a major version number only by committing a (pious?) fraud.

Comment (cdata) [inline iron-iconset-svg.html]: SGTM

**Comment by cdata** [comment]
LGTM :+1:

**Comment by jurgenfink** [comment]
Found the solution.

The cause of the warning in Chrome debugger is due to the wrong timing of loading the link imports in the right sequence.

**Solution:**

 **1.) Remove link imports in the iron-icons** (and other icon-sets if needed, like maps, social, etc ...):

*  public
  *  bower_components
     * iron-icons
        *  **iron-icons.html**
        *  **maps-icons.html** (optional, if you are using them)
        *  **social-icons.html** (optional, if you are using them)

**iron-icons.html:**

before:

    
    <link rel='import' href='../iron-icon/iron-icon.html'>
    <link rel='import' href='../iron-iconset-svg/iron-iconset-svg.html'>
    
    <iron-iconset-svg name="icons" size="24">
        <svg><defs>
        <g id="3d-rotation"><path d="M7.52 21.48C ..... etc ..... /></g>
        </defs></svg>
    </iron-iconset-svg>

after:

    
    
    
    <iron-iconset-svg name="icons" size="24">
        <svg><defs>
        <g id="3d-rotation"><path d="M7.52 21.48C ..... etc ..... /></g>
        </defs></svg>
    </iron-iconset-svg>

The initial link-imports (dependencies) are blocking (not loading async but sync, which is good because that's the way it should be). 
However, within the code of `<link rel='import' href='../iron-icon/iron-icon.html'>`  iron-icon is making reference to the icon set that could not load yet (`<iron-iconset-svg name="icons" size="24">` etc ...) because it comes after that initial link-import (due to blocking nature of link import). Hence, at line 164 it cannot find the Default iconset **icons**, and therefore throwing the famous warning at line 167:

> could not find iconset icons, did you import the iconset?

**2.) Load required dependencies** in your project file in the correct sequence:

    <head>
      <meta charset="utf-8" />
      <title></title>
    
      <script src="/bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
      <link rel="import" href="/bower_components/polymer/polymer.html">
    
      <link rel="import" href="/bower_components/iron-meta/iron-meta.html">
      <link rel="import" href="/bower_components/iron-flex-layout/iron-flex-layout.html">
      <link rel="import" href="/bower_components/iron-iconset-svg/iron-iconset-svg.html">
      <link rel="import" href="/bower_components/iron-iconset/iron-iconset.html">
    
      <link rel="import" href="/bower_components/iron-icons/iron-icons.html">
      <link rel="import" href="/bower_components/iron-icons/maps-icons.html">
      <link rel="import" href="/bower_components/iron-icons/social-icons.html">
    
      <link rel="import" href="/bower_components/iron-icon/iron-icon.html">
    </head>

The `<link rel="import" href="/bower_components/iron-icon/iron-icon.html">` is loaded in last position now, hence all dependencies are available at this point.

Works like a charm for me now.

**Pull request merged**